ivy.el

```diff
@@ -2827,6 +2827,35 @@ Possible choices are 'ivy-magic-slash-non-match-cd-selected,
                   magic)
              (ivy--create-and-cd canonical))))))
 
+(defcustom ivy-magic-dollar t
+  "When non-nil, $ will read an file path environment variable
+and goto its value."
```

Comment:
Nitpick: Please follow the conventions outlined in [`(elisp) Documentation Tips`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html) for this docstring and the one below. In particular, please avoid making the summary a multi-line sentence.

Reply:
done